DockerHub/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "Non Serviam",
+    "id": "154651965022928896"
+  },
+  "url": "hub.docker.com",
+  "description": {
+    "en": "Docker Hub is a service provided by Docker for finding and sharing container images with your team."
+  },
+  "service": "DockerHub",
+  "version": "1.0",
+  "logo": "https://i.imgur.com/xr2Gulz.png",
+  "thumbnail": "https://i.imgur.com/n3Ox5DQ.png",
+  "color": "#099CEC",
+  "tags": [
+    "docker"
+  ],
+  "category": "other",
+  "regExp": "hub\\.docker\\.com"
```

Comment:
Remove this, the url will do.

Reply:
Was thinking initially on giving presence to the forum and/or the docs and forgot to remove the regex